### PR TITLE
fix: web manifest update

### LINKED ISSUE #1998: 🐛 Bug Report: manifest error in Chrome
### 📜 Description

When I go to web.novu.co main page and I am authorized I receive an error message:
demo

### 👟 Reproduction steps

1. Go to [site](https://web.novu.co).
2. Login.
3. Open console.

### 👍 Expected behavior

Icons should exist.

### 👎 Actual Behavior with Screenshots

demo

### 💻 Operating system

MacOS

### 🤖 Node Version

18.11.0

### 📃 Provide any additional context for the Bug.

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by MaximFedarau**
### What change does this PR introduce? 
Fix for #1998.
I implemented @p-fernandez suggestion.

### Why was this change needed?
Closes #1998.

### CODE REVIEW

File: apps/web/public/manifest.json [hunk 1]
```diff
@@ -6,16 +6,6 @@
       "src": "favicon.ico",
```
Comment (LetItRock) [inline apps/web/public/manifest.json:6]: I feel like we should rather remove this file completely or update it with the appropriate information. WDYT? @novuhq/novu-devs

**Comment by MaximFedarau** [comment]
Thanks @LetItRock for your review! So what should I do: remove all these icons, remove icons, that Novu doesn't have or add them?

File: apps/web/public/manifest.json (hunk 1 above)
Comment (p-fernandez) [reply apps/web/public/manifest.json:6]: Yeah, @MaximFedarau update the `short_name` and `name` to:
```
  "short_name": "Novu Web",
  "name": "Novu Web application",``
```
for example

**Review by p-fernandez — CHANGES_REQUESTED** [review]

File: apps/web/public/manifest.json (hunk 1 above)
Comment (MaximFedarau) [reply apps/web/public/manifest.json:6]: Thanks, @LetItRock and @p-fernandez. Done.

**Comment by MaximFedarau** [comment]
@p-fernandez, @LetItRock, implemented your suggestions.

**Review by p-fernandez — APPROVED** [review]

**Comment by MaximFedarau** [comment]
@p-fernandez, please, can you restart failed tests?

**Comment by MaximFedarau** [comment]
@p-fernandez, @ainouzgali , can it be merged?

**Pull request merged**